iOS-ObjC/MSALiOSobjC/ViewController.m

```diff
@@ -0,0 +1,278 @@
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//------------------------------------------------------------------------------
+
+
+#import "ViewController.h"
+#import <MSAL/MSAL.h>
+
+/// 😃 A View Controller that will respond to the events of the Storyboard.
+
+@interface ViewController ()
+    
+    @end
+
+@implementation ViewController
+    
+    MSALPublicClientApplication *application;
+    
+    // Update the below to your client ID you received in the portal. The below is for running the demo only
+    NSString *clientId = @"66855f8a-60cd-445e-a9bb-8cd8eadbd3fa";
+    // Additional variables for Auth and Graph API
+//    NSString *redirectUri = @"msalad952fe6-c650-467b-abd6-def8f2c28a8e://auth";
+    NSString *authorityString = @"https://login.microsoftonline.com/common";
+    NSString *graphURI = @"https://graph.microsoft.com/v1.0/me/";
+    
+    /**
+     Setup public client application in viewDidLoad
+     */
+    
+    - (void)viewDidLoad {
+        [super viewDidLoad];
+        // Do any additional setup after loading the view.
+        [self initMSAL];
+    }
+    
+    // MARK: Initialization
+
+    -(void) initMSAL{
+        
+        /**
+         
+         Initialize a MSALPublicClientApplication with a given clientID and authority
+         
+         - clientId:            The clientID of your application, you should get this from the app portal.
+         - redirectUri:         A redirect URI of your application, you should get this from the app portal.
+         If nil, MSAL will create one by default. i.e./ msauth.<bundleID>://auth
+         - authority:           A URL indicating a directory that MSAL can use to obtain tokens. In Azure AD
+         it is of the form https://<instance/<tenant>, where <instance> is the
+         directory host (e.g. https://login.microsoftonline.com) and <tenant> is a
+         identifier within the directory itself (e.g. a domain associated to the
+         tenant, such as contoso.onmicrosoft.com, or the GUID representing the
+         TenantID property of the directory)
+         - error                The error that occurred creating the application object, if any, if you're
+         not interested in the specific error pass in nil.
+         */
+        
+        NSError *error = nil;
+        
+        NSURL *authorityURL = [NSURL URLWithString:(authorityString)];
```

Comment:
no need for curly braces around authorityString

Reply:
removed curly braces there